bin/gg

```diff
@@ -20,7 +20,12 @@ switch (cli.command) {
 	case 'cl':
 	case 'clone':
 		var repository = cli.args[0];
-		gg.clone(repository);
+
+		if (repository.substring(0, 7) == 'http://') { // URL.
```

Comment:
@qw3rtman Wouldn't this change wrap the SSH form too, e.g. `https://github.com/git@github.com:<username>/<repo>.git.git`, since it seems to make that decision based solely on not detecting an `https://` prefix?